diff --git a/android_webview/lib/BUILD.gn b/android_webview/lib/BUILD.gn
--- a/android_webview/lib/BUILD.gn
+++ b/android_webview/lib/BUILD.gn
@@ -24,6 +24,8 @@
     "//android_webview/js_sandbox/service:js_sandbox",
     "//android_webview/renderer",
     "//base",
+    "//chrome/browser:browser",
+    "//chrome/utility",
     "//cc/base",
     "//components/autofill/core/common",
     "//components/crash/core/common",
